back/spec/acceptance/baskets_ideas_spec.rb

```diff
@@ -0,0 +1,140 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource BasketsIdea do
+  explanation 'Ideas included in a basket, with the count of votes.'
+
+  before do
+    header 'Content-Type', 'application/json'
+    @user = create(:user)
+    @project = create(:continuous_single_voting_project)
+    @basket = create(:basket, participation_context: @project, user: @user)
```

Comment:
I think the main reason I did this is because I don't think you can have global lets. Also we always set the @user in tests so we can set the header token. Maybe I can try to put the project and baskets in a "global" let?

Reply:
I replaced the instance variables by lets, and it worked fine. For some reason, I thought it would not be possible 😅 I feel like the spec file is much cleaner like this indeed.